### PR TITLE
Dummy links for unknown links

### PR DESCRIPTION
**Pull request by shivamsaraf**

### CODE REVIEW

File: src/Components/Header.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Header extends React.Component {
             </a>
           </div>
           <div className="flex-container2">
-            <div className="L1">Career Development Cell</div>
+            <div className="L1">Career Development Caell</div>
```
Comment (sanketsingh24) [inline src/Components/Header.js]: Please fix this

File: src/Pages/Company/RegistrationAndBrochure.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ class RegistrationAndBrochure extends React.Component {
                 <hr />
               </div>
               <div className="btn" align="middle">
-                <a href="https://www.iitism.ac.in/pdfs/tnp/Placement-Brochure.pdf" target="_blank">
+                <a href="https://www.iitism.ac.in/pdfs/tnp/Placement-Brochure.pdf" target="_blank" rel="noopener noreferrer">
```
Comment (sanketsingh24) [inline src/Pages/Company/RegistrationAndBrochure.js]: `noreferref` automatically contains the functionality of `noopener`, so it is not required.

File: src/Components/Header.js (hunk 1 above)
Comment (shivamsaraf) [reply src/Components/Header.js]: Fixed it and sent PR again

File: src/Pages/Company/RegistrationAndBrochure.js (hunk 2 above)
Comment (shivamsaraf) [reply src/Pages/Company/RegistrationAndBrochure.js]: Corrected

**Pull request merged**